torch/_subclasses/meta_utils.py

```diff
@@ -476,6 +487,64 @@ def as_json(self, describer_id: _DescriberId) -> Dict[str, object]:
         }
 
 
+@dataclass(frozen=True)
+class _ViewFunc(Generic[_TensorT]):
```

Comment:
FWIW I don't really see why this should be private, if MetaTensorDesc isn't considered private per this file.

Reply:
It doesn't have to be - but it's kind of an implementation detail...